### PR TITLE
RAC-485: Add simple select in DSM

### PR DESCRIPTION
**Pull request by StevenVAIDIE**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: akeneo-design-system/src/components/Input/SelectInput/SelectInput.tsx [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+`;
+
+const OptionContainer = styled.div`
+  background: ${getColor('white')};
+  height: 34px;
+  padding: 0 20px;
+  align-items: center;
+  gap: 10px;
+  cursor: pointer;
+
```
Comment (ValentinMumble) [inline akeneo-design-system/src/components/Input/SelectInput/SelectInput.tsx:54]:
```suggestion
```

File: akeneo-design-system/src/components/Input/SelectInput/SelectInput.tsx [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+  color: ${getColor('grey', 120)};
+  line-height: 34px;
+
+  &:focus {
+    color: ${getColor('grey', 120)};
+  }
+  &:hover {
+    background: ${getColor('grey', 20)};
+  }
+  &:hover {
```
Comment (ValentinMumble) [inline akeneo-design-system/src/components/Input/SelectInput/SelectInput.tsx:67]:
```suggestion
```

File: akeneo-design-system/src/components/Input/SelectInput/SelectInput.tsx [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+  &:active {
+    color: ${getColor('brand', 100)};
+    font-weight: 700;
+  }
+  &:disabled {
+    color: ${getColor('grey', 100)};
+  }
+`;
+
+const EmptyResultContainer = styled.div<{tall: boolean} & AkeneoThemedProps>`
```
Comment (ValentinMumble) [inline akeneo-design-system/src/components/Input/SelectInput/SelectInput.tsx:79]:
```suggestion
const EmptyResultContainer = styled.div`
```

File: akeneo-design-system/src/components/Input/SelectInput/SelectInput.tsx [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+
+    /**
+     * Force the vertical position of the overlay.
+     */
+    verticalPosition?: VerticalPosition;
+  }
+>;
+
+/**
+ * Select input allows the user to select content and data when the expected user input is composed of one.
```
Comment (ValentinMumble) [inline akeneo-design-system/src/components/Input/SelectInput/SelectInput.tsx:188]: Of one what?

File: akeneo-design-system/src/components/Input/index.ts [hunk 5]
```diff
@@ -2,4 +2,5 @@ export * from './BooleanInput/BooleanInput';
 export * from './InputProps';
 export * from './NumberInput/NumberInput';
 export * from './TextInput/TextInput';
+export * from './SelectInput/SelectInput';
```
Comment (ValentinMumble) [inline akeneo-design-system/src/components/Input/index.ts:5]: Just one line above plz

**Review by ValentinMumble — APPROVED** [review]

File: akeneo-design-system/src/components/Input/SelectInput/SelectInput.tsx (hunk 4 above)
Comment (StevenVAIDIE) [reply akeneo-design-system/src/components/Input/SelectInput/SelectInput.tsx]: :metal:

**Review by juliensnz — APPROVED** [review]

**Pull request merged**